pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/BaseResource.java

```diff
@@ -282,4 +282,21 @@ public static String getReasonFromServer(WebApplicationException e) {
             }
         }
     }
+
+    public <T> CompletableFuture<T> getCompletableFuture(WebTarget path) {
```

Comment:
I think this method needs a better name to indicate what it is actually doing. It's probably sufficient to overload the `asyncGetRequest` method name here and make its signature: 

```java
public <T> CompletableFuture<T> getCompletableFuture(WebTarget path)
```

Reply:
@michaeljmarshall hello，Why not reply to me, I have been waiting for you to reply to the message, but I have not answered, I am very anxious, the optimization does not conform to the open source standards, if there is not meet the norms, please make suggestions, I hope to be able to participate.

Reply:
Looks good to me. Sorry, again, for my delayed review.